src/importEndpoint.js

```diff
@@ -0,0 +1,202 @@
+		(at your option) any later version.
+		
+		Zotero is distributed in the hope that it will be useful,
+		but WITHOUT ANY WARRANTY; without even the implied warranty of
+		MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	See the
+		GNU Affero General Public License for more details.
+		
+		You should have received a copy of the GNU Affero General Public License
+		along with Zotero.	If not, see <http://www.gnu.org/licenses/>.
+		
+		***** END LICENSE BLOCK *****
+*/
+
+const config = require('config');
+const Translate = require('./translation/translate');
+
+const fields = {
+	valid: [
+		"itemType",
+		"dateAdded",
+		"dateModified",
+		"tags",
+		"attachments",
+		"notes",
+		"collections",
+		"url",
+		"rights",
+		"series",
+		"volume",
+		"edition",
+		"place",
+		"publisher",
+		"ISBN",
+		"date",
+		"callNumber",
+		"archiveLocation",
+		"extra",
+		"accessDate",
+		"seriesNumber",
+		"numberOfVolumes",
+		"libraryCatalog",
+		"language",
+		"abstractNote",
+		"title",
+		"shortTitle",
+		"numPages",
+		"archive",
+		"pages",
+		"publicationTitle",
+		"issue",
+		"ISSN",
+		"journalAbbreviation",
+		"DOI",
+		"seriesTitle",
+		"seriesText",
+		"section",
+		"type",
+		"medium",
+		"runningTime",
+		"artworkSize",
+		"number",
+		"code",
+		"session",
+		"legislativeBody",
+		"history",
+		"reporter",
+		"court",
+		"committee",
+		"assignee",
+		"priorityNumbers",
+		"references",
+		"legalStatus",
+		"country",
+		"applicationNumber",
+		"issuingAuthority",
+		"filingDate",
+		"codeNumber",
+		"scale",
+		"meetingName",
+		"versionNumber",
+		"system",
+		"programmingLanguage",
+		"conferenceName"
+	],
+
+	aliasOf: {
+		bookTitle: "publicationTitle",
+		thesisType: "type",
+		university: "publisher",
+		letterType: "type",
+		manuscriptType: "type",
+		interviewMedium: "medium",
+		distributor: "publisher",
+		videoRecordingFormat: "medium",
+		genre: "type",
+		artworkMedium: "medium",
+		websiteType: "type",
+		websiteTitle: "publicationTitle",
+		institution: "publisher",
+		reportType: "type",
+		reportNumber: "number",
+		billNumber: "number",
+		codeVolume: "volume",
+		codePages: "pages",
+		dateDecided: "date",
+		reporterVolume: "volume",
+		firstPage: "pages",
+		caseName: "title",
+		docketNumber: "number",
+		documentNumber: "number",
+		patentNumber: "number",
+		issueDate: "date",
+		dateEnacted: "date",
+		publicLawNumber: "number",
+		nameOfAct: "title",
+		subject: "title",
+		mapType: "type",
+		blogTitle: "publicationTitle",
+		postType: "type",
+		forumTitle: "publicationTitle",
+		audioRecordingFormat: "medium",
+		label: "publisher",
+		presentationType: "type",
+		studio: "publisher",
+		network: "publisher",
+		episodeNumber: "number",
+		programTitle: "publicationTitle",
+		audioFileType: "medium",
+		company: "publisher",
+		proceedingsTitle: "publicationTitle",
+		encyclopediaTitle: "publicationTitle",
+		dictionaryTitle: "publicationTitle"
+	}
+};
```

Comment:
Remove — we don't want hard-coded fields in here.

Reply:
How do I get these dynamically in the translation-server?

Reply:
Why do you need them?

Reply:
On import, a translator could offer either one of field or basefield, and Zotero would do the right thing on save. I need to be able to do the right thing to mimic this going either way.

Reply:
Zotero also strips invalid fields on save if a translator offers them.

Reply:
But the tests should reflect the output of the translator as generated via Scaffold. The output shouldn't need to be modified. They're not testing what you get on save — they're testing the literal output of the translator.

Reply:
Alright.